### PR TITLE
Feature/check network state

### LINKED ISSUE #103: Init blockchain when no internet connection
Handle broken connection and no connection with  the blockchain

### PR DESCRIPTION
**Pull request by jonathanraes**
Closes #103 
Checks network state in SplashActivity

### CODE REVIEW

**Comment by landgenoot** [comment]
Add snackbar with shortcut to connection settings.

**Review by wkmeijer — CHANGES_REQUESTED** [review]
When you re-enable the network and go back to the app too fast the toast that says "Please enable network connection" is shown, even though the network connection was just enabled.

When the network connection is disabled in e.g. mainactivity no error is given, even though it should still update the blockchain.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -76,6 +76,11 @@
     <string name="shared_preferences_key_election">com.digitalvotingpass.ELECTION_OBJECT_KEY</string>
     <string name="shared_preferences_file">com.digitalvotingpass.shared</string>
 
+    <string name="please_enable_connect_message">Please enable network connection</string>
+    <string name="no_connection">No Network</string>
+
+    <string name="go_network_settings">Settings</string>
```
Comment (wkmeijer) [inline app/src/main/res/values/strings.xml]: Please change this to "WIFI SETTINGS" so it's more clear where the button will take the users.

**Review by wkmeijer — CHANGES_REQUESTED** [review]
Snackbar is an improvement. Handling lost network connection in the rest of the app will be done at a later point. 

Please also run 'optimize imports' on the project one last time.

**Comment by jonathanraes** [comment]
waiting for #139 for updated translations,

**Review by wkmeijer — APPROVED** [review]

**Pull request merged**